packages/client/package.json

```diff
@@ -9,47 +9,17 @@
   "engines": {
     "node": "14"
   },
-  "dependencies": {
-    "@amsterdam/asc-assets": "^0.25.1",
-    "@amsterdam/asc-ui": "^0.25.1",
-    "@apollo/client": "^3.1.3",
-    "@datapunt/matomo-tracker-react": "^0.2.1",
-    "@sentry/browser": "^5.20.0",
-    "@types/lodash.isboolean": "^3.0.6",
-    "@types/lodash.isnumber": "^3.0.6",
-    "@types/lodash.isstring": "^4.0.6",
-    "dotenv-flow": "^3.1.0",
-    "eslint-plugin-flowtype": "^5.1.3",
-    "feature-policy": "^0.5.0",
-    "graphql": "^14.7.0",
-    "graphql.macro": "^1.4.2",
-    "history": "^5.0.0",
-    "lodash.isboolean": "^3.0.3",
-    "lodash.isnumber": "^3.0.3",
-    "lodash.isstring": "^4.0.1",
-    "named-urls": "^2.0.0",
-    "react": "^16.14.0",
-    "react-app-polyfill": "^1.0.6",
-    "react-device-detect": "^1.12.1",
-    "react-dom": "^16.14.0",
-    "react-helmet": "^6.1.0",
-    "react-hook-form": "^6.0.0",
-    "react-markdown": "^4.3.1",
-    "react-router-dom": "^5.2.0",
-    "slugify": "^1.4.0",
-    "styled-components": "^5.1.1"
-  },
   "scripts": {
-    "prestart": "npm install",
-    "start": "REACT_APP_GIT_SHA=`git rev-parse --short HEAD` REACT_APP_GIT_BRANCH=`git rev-parse --abbrev-ref HEAD` react-scripts start",
+    "_prestart": "npm install",
```

Comment:
This has been done on purpose right? Because `npm start` fails with the prestart.

Reply:
@afjlambert

Reply:
Yeah, let's discuss in next dev-meeting how we want to resolve this. I'll add it to the agenda.